RecoMET/METPUSubtraction/plugins/PFMETProducerMVA.cc

```diff
@@ -16,12 +16,12 @@ PFMETProducerMVA::PFMETProducerMVA(const edm::ParameterSet& cfg)
   srcPFCandidatesView_ = consumes<reco::CandidateView>(cfg.getParameter<edm::InputTag>("srcPFCandidates"));
   srcVertices_     = consumes<reco::VertexCollection>(cfg.getParameter<edm::InputTag>("srcVertices"));
   vInputTag srcLeptonsTags = cfg.getParameter<vInputTag>("srcLeptons");
+  consumes<double>(edm::InputTag("fixedGridRhoFastjetAll"));
```

Comment:
So this took me the longest. Actually I don't really know what really is 
consuming this; it was just saved and used for nothing. So this line is 
just a fix to stop CMSSW complaining about it. When making it 
configurable a user could think that it actually changes the result, 
which is not the case. So if you want to make it configurable we need a 
big warning what this parameter is for.

Cheers,
Raphael

On 19.01.2016 16:55, mmarionncern wrote:
>
> In RecoMET/METPUSubtraction/plugins/PFMETProducerMVA.cc 
> <https://github.com/cms-met/cmssw/pull/9#discussion_r50129908>:
>
> > @@ -16,12 +16,12 @@ PFMETProducerMVA::PFMETProducerMVA(const edm::ParameterSet& cfg)
> >    srcPFCandidatesView_ = consumes<reco::CandidateView>(cfg.getParameter<edm::InputTag>("srcPFCandidates"));
> >    srcVertices_     = consumes<reco::VertexCollection>(cfg.getParameter<edm::InputTag>("srcVertices"));
> >    vInputTag srcLeptonsTags = cfg.getParameter<vInputTag>("srcLeptons");
> > +  consumes<double>(edm::InputTag("fixedGridRhoFastjetAll"));
>
> Why hardcoding it?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-met/cmssw/pull/9/files#r50129908>.
>


-- 
Raphael Friese

Karlsruhe Institute of Technology (KIT)
Kaiserstraße 12
Building 30.23 / R 8-22
76131 Karlsruhe, Germany
Phone: +49 721 608-47243

CERN
Building 32-4 B06
Phone: +41 22 76-78267

Email: Raphael.Friese@cern.ch